CMakeLists.txt

```diff
@@ -9,14 +9,14 @@ option(FAASM_SELF_TRACING "Turn on system tracing using the logger" OFF)
 option(FAASM_PERF_PROFILING "Turn on profiling features as described in debugging.md" OFF)
 
 # WAMR and interpreter
-option(WAMR_INTERPRETER_MODE "Turns interpreter on / off (vs. AoT)" OFF)
+option(FAASM_WAMR_AOT_MODE "Turns on AoT mode in WAMR" ON)
```

Comment:
Now the behaviour for SGX and non-SGX WAMR in terms of options is the same:
* Interpreter mode is always built.
* If opt-in, AoT support is also built.
* If both available, will run in AoT mode.

Reply:
As discussed offline, I think we should remove interpreter completely.

Reply:
Agree. Will do it in a separate PR.